scripts/docgen.ts

```diff
@@ -12,6 +12,7 @@ const EXTRA_FILES_PATHS = [
   '../src/mantine-core/src/components/Timeline/TimelineItem/TimelineItem.tsx',
   '../src/mantine-core/src/components/Slider/Slider/Slider.tsx',
   '../src/mantine-core/src/components/Slider/RangeSlider/RangeSlider.tsx',
+  '../src/mantine-core/src/components/Stepper/Step/Step.tsx',
```

Comment:
I added this extra file for docgen because otherwise the `Step Component props` section in `Stepper` props tabs is empty.